iso4217-historical.yml

```diff
@@ -0,0 +1,99 @@
+---
+BYR:
+  minor_unit: 0
+  currency: Belarussian Ruble
+  symbol: p.
+  supported_providers:
+  - :oxr
+  :from_date: '2000-01-01'
```

Comment:
Any reason these are symbol keys rather than string? Seems odd having them different than the other keys.

Reply:
Actually, I think that was a side effect of having edited some of the currencies in irb and saving them - which has then been propagated out. I'll fix them up